src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -1115,12 +1115,10 @@ namespace ReportSizingManager {
 					}
 				} else if ( SizingType == HeatingWaterDesCoilLoadUsedForUASizing ) {
 					if ( TermUnitSingDuct || TermUnitPIU || TermUnitIU ) {
-						DesMassFlow = StdRhoAir * TermUnitSizing( CurZoneEqNum ).AirVolFlow * TermUnitSizing( CurZoneEqNum ).ReheatAirFlowMult;
```

Comment:
See how hard it is to interpret the code here!

Because DesMassFlow is not used within this IF block of the next ELSE IF. It is needed in the ELSE.

    } else if ( SizingType == HeatingWaterDesCoilLoadUsedForUASizing ) {
        if ( TermUnitSingDuct || TermUnitPIU || TermUnitIU ) {
            Cp = GetSpecificHeatGlycol( PlantLoop( DataWaterLoopNum ).FluidName, DataGlobals::HWInitConvTemp, PlantLoop( DataWaterLoopNum ).FluidIndex, CallingRoutine );
            rho = GetDensityGlycol( PlantLoop( DataWaterLoopNum ).FluidName, DataGlobals::CWInitConvTemp, PlantLoop( DataWaterLoopNum ).FluidIndex, CallingRoutine );
            AutosizeDes = DataWaterFlowUsedForSizing * DataWaterCoilSizHeatDeltaT * Cp * rho * TermUnitSizing( CurZoneEqNum ).ReheatLoadMult;
        } else if ( ZoneEqFanCoil || ZoneEqUnitHeater ) {
            Cp = GetSpecificHeatGlycol( PlantLoop( DataWaterLoopNum ).FluidName, DataGlobals::HWInitConvTemp, PlantLoop( DataWaterLoopNum ).FluidIndex, CallingRoutine );
            rho = GetDensityGlycol( PlantLoop( DataWaterLoopNum ).FluidName, DataGlobals::CWInitConvTemp, PlantLoop( DataWaterLoopNum ).FluidIndex, CallingRoutine );
            AutosizeDes = DataWaterFlowUsedForSizing * DataWaterCoilSizHeatDeltaT * Cp * rho;
        } else {
            DesMassFlow = FinalZoneSizing( CurZoneEqNum ).DesHeatMassFlow;
            CoilInTemp = FinalZoneSizing( CurZoneEqNum ).DesHeatCoilInTemp;
            CoilInHumRat = FinalZoneSizing( CurZoneEqNum ).DesHeatCoilInHumRat;
            CoilOutTemp = FinalZoneSizing( CurZoneEqNum ).HeatDesTemp;
            CoilOutHumRat = FinalZoneSizing( CurZoneEqNum ).HeatDesHumRat;
            AutosizeDes = PsyCpAirFnWTdb( CoilOutHumRat, 0.5 * ( CoilInTemp + CoilOutTemp ) ) * DesMassFlow * ( CoilOutTemp - CoilInTemp );
        }
    } else if ( SizingType == HeatingWaterDesCoilWaterVolFlowUsedForUASizing ) {